backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/bpartner/service/impl/BPBankAccountDAO.java

```diff
@@ -140,6 +141,24 @@ else if (query.getInvoiceId() != null)
 							.addEqualsFilter(I_C_Invoice.COLUMNNAME_C_Invoice_ID, query.getInvoiceId())
 							.create());
 		}
+
+		if (Check.isNotBlank(query.getDescription()))
+		{
+			queryBuilder.addStringLikeFilter(I_C_BP_BankAccount.COLUMNNAME_Description, query.getDescription(), true);
+		}
```

Comment:
- I don't think it makes much sense to make this lookup, I mean, `description` is not reliable enough to do lookups